Tasks/Gradle/CodeAnalysis/Common/AnalysisResult.ts

```diff
@@ -1,11 +1,13 @@
+import {IAnalysisTool} from './IAnalysisTool';
+
 /**
  * Data class that describes code analysis results
  * 
  * @export
  * @class AnalysisResult
  */
 export class AnalysisResult {
-    constructor(public toolName: string, public moduleName: string, public resultFiles: string[], public violationCount: number, public affectedFileCount: number) {
+    constructor(public originatingTool: IAnalysisTool, public moduleName: string, public resultFiles: string[], public violationCount: number, public affectedFileCount: number) {
```

Comment:
In the old architecture there was a capability to check if a tool was enabled by its name, which has since been deprecated in favour of a full object representation (IAnalysisTool). This changeset updates AnalysisResult accordingly.